test/features/buckets/revisioned_bucket.js

```diff
@@ -6,6 +6,8 @@ var server = require('../../utils/server.js');
 var uuid = require('cassandra-uuid').TimeUuid;
 var P = require('bluebird');
 
+var mwUtils = require('../../../lib/mwUtil');
```

Comment:
Those relative paths are a bit awkward.. One interesting pattern I have seen recently is to link a module into its own node_modules, making it easier to require (as `restbase/mwUtil`). Perhaps worth considering when / if we start using babel & a build step.